.github/workflows/pages.yaml

```diff
@@ -14,19 +14,26 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Configure pages
+        uses: actions/configure-pages@v5
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+          cache: "pip"
+
+      - name: Install dependencies
+        run: pip install .[docs]
```

Comment:
The `.[docs]` here means we install all the base dependencies _and_ the `docs` dependencies from `pyproject.toml`. Pretty nifty.